<?xml version="1.0"?>
<launch>
<node name="SinusoidalBLDC" pkg="sinusoidal" type="motordriver.py" output="screen">
<param name="publish_tf" value="false"/>
<param name="publish_odom" value="false"/>
</node>
</launch>
